doc/website/getting-started/overview.md

```diff
@@ -95,7 +94,7 @@ while (keepRunning)
 By calling `take` we get an `expected` and hence we have to handle the potential error.
 
 And that's it. We have created our first simple iceoryx example.
-[Here](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/README.md) you can find further examples
+[Here](../../../iceoryx_examples/README.md) you can find further examples
```

Comment:
We could here link to `../examples/`.

Reply:
I would not do this since all the examples files are just stub documents where only the title is mentioned and some weird broken relative link.